app/parsers/ted.py

```diff
@@ -156,6 +163,7 @@ def extract_data(archive_path, extract_path):
             # Return the name of the folder containing the extracted files
             return tf.getnames()[0].split("/")[0]
         except (EOFError, FileNotFoundError) as e:
+            logging.warning('maybe in line 165')
```

Comment:
Leftover debugging log?

Reply:
This line should be deleted. I will fix it.